include/zephyr/bluetooth/conn.h

```diff
@@ -98,9 +98,12 @@ enum {
 
 /** Preferred PHY parameters for LE connections */
 struct bt_conn_le_phy_param {
-	uint16_t options;     /** Connection PHY options. */
-	uint8_t  pref_tx_phy; /** Bitmask of preferred transmit PHYs */
-	uint8_t  pref_rx_phy; /** Bitmask of preferred receive PHYs */
+	/** Connection PHY options. */
+	uint16_t options;
```

Comment:
Non-blocking but if you prefer you may want to use the following syntax (i.e. tell doxygen comment block applies to previous element)

```suggestion
	uint16_t options;     /**< Connection PHY options. */
```